UNRELEASED.md

```diff
@@ -1,3 +1,12 @@
+**Breaking:**
+Removing deprecated border tokens and mixins.
```

Comment:
Maybe a short guide for what people should do instead?

We wrote this in the past when we deprecated them:
```
- Deprecate bpk-border-radius-pill token. Use bpk-border-radius-md instead. Otherwise, check with your designer to choose any other supported radius.
- Deprecate bpk-border-radius-pill-lg token. Use bpk-border-radius-lg instead.